File: app/src/main/java/zachwhaley/atwork/BootReceiver.java
package zachwhaley.atwork;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        AtWork.log("BootReceiver.onRecieve");
        if (intent.getAction().equals("android.intent.action.BOOT_COMPLETED")) {
            AtWork.setAlarm(context);
            AtWork.sendNotification(context, "Boot receiver");
        }
    }
}

File: app/src/main/java/zachwhaley/atwork/BootReceiver.kt
package zachwhaley.atwork

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        AtWork.log("BootReceiver.onReceive")
        if (intent.action == "android.intent.action.BOOT_COMPLETED") {
            AtWork.setAlarm(context)
        }
    }
}
